core/chains/evm/logpoller/log_poller.go

```diff
@@ -439,17 +439,18 @@ func (lp *logPoller) run() {
 			fromBlock, err := lp.GetReplayFromBlock(lp.ctx, fromBlockReq)
 			if err == nil {
 				if !filtersLoaded {
-					lp.lggr.Warnw("Received replayReq before filters loaded", "fromBlock", fromBlock, "requested", fromBlockReq)
+					lp.lggr.Warnw("received replayReq before filters loaded", "fromBlock", fromBlock, "requested", fromBlockReq)
```

Comment:
Log messages can (should?) start with capital letters

Reply:
Oh, weird... for some reason when I tried adding an error starting with lowercase it gave me a warning saying "all error messages should be lowercase and have no punctuation".  I figured it was coming from golint or gofmt, but maybe it's a builtin GoLand thing?  I'll check my settings.